pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[project]
+name = "lti-emailer"
+version = "2.11"
+requires-python = ">=3.10,<3.11"
+dependencies = [
+    "canvas-python-sdk",
+    "oracledb==1.1.1",
+    "django==3.2.16",
+    "django-allow-cidr==0.5.0",
+    "django-angular==2.3",
+    "django-auth-lti",
+    "django-cached-authentication-middleware==0.2.2",
+    "django-canvas-lti-school-permissions",
+    "django-coursemanager",
+    "django-icommons-common",
+    "django-icommons-ui",
+    "django-redis-cache==3.0.1",
+    "django-ssm-parameter-store",
+    "django-storages==1.12.3",
+    "django-watchman==1.3.0",
+    "flanker==0.9.11",
+    "harvard-django-utils",
+    "hiredis==2.0.0",
+    "psycopg2==2.9.3",
+    "python-json-logger==2.0.4",
+    "redis==3.5.3",
+    "requests==2.28.1",
+    "splunk-handler==3.0.0",
+]
+
+[tool.ruff.lint]
+ignore = ["F403", "F405", "F841", "E741"]  # Globally ignore star-import warnings, unused vars, and ambiguous names
+
+# TODO: Once TLT-5255 is completed. Edit the dependencies in the pyproject.toml file:
+# for any libraries that were installed directly from GitHub,
```

Comment:
Make sure to track this + project requirement upgrades as part of a tech debt story

Reply:
Good idea, I had created this story ([TLT-5256](https://at-harvard.atlassian.net/browse/TLT-5256)) to track that. Let me know if I should add some other notes in the description and feel free to also modify it.